website/src/components/CalcSpectrum.tsx

```diff
@@ -60,6 +63,7 @@ export const CalcSpectrum: React.FC = () => {
     useState<CalcSpectrumPlotData | undefined>(undefined);
   //state
   const [isNonEquilibrium, setIsNonEquilibrium] = useState<boolean>(false);
+  const [isGeisa, setisGeisa] = useState<boolean>(false);
```

Comment:
```suggestion
  const [useGesia, setUseGesia] = useState<boolean>(false);
```
I think "use" would be a better word for indicating to use the gesia databank.